android-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 1.1.9 (Nov 21, 2019)
 * Enable to set location, ip address, and carrier on a MolocoView and a MolocoNative.
+* Fix a bug on the Webview scaling for Android API (< 19).
```

Comment:
nit: IP, WebView?

Reply:
I revised it on 6bf4663.
Thank you for the comment!